### PR TITLE
 Properly quote rofi call in i3 config

### PR DESCRIPTION
**Pull request by ImranVirani**

### CODE REVIEW

**Comment by orestisfl** [comment]
What error do you get? It works fine for me.

**Comment by Airblader** [comment]
Seems fair, though, rofi's manpage also lists it as a comma separated list.

**Comment by orestisfl** [comment]
> Seems fair, though, rofi's manpage also lists it as a comma separated list.

Replacing the comma makes the `run` menu not show up in my machine.

**Comment by Airblader** [comment]
Maybe the command needs to be quoted?

**Comment by ImranVirani** [comment]
Exact error is:

ERROR: Received ConfigureNotify for unknown window 0x00600000
ERROR: Received ExposeEvent for unknown window 0x00600000
ERROR: Received ConfigureNotify for unknown window 0x00600000
ERROR: Expected one of these tokens: <end>, '[', 'move', 'exec', 'restart', 'reload', 'shmlog', 'debuglog', 'border', 'layout', 'append_layout', 'workspace', 'focus', 'kill', 'open', 'fullscreen', 'stick', 'split', 'floating', 'mark', 'unmark', 'resize', 'rename', 'nop', 'scratchpad', 'swap', 'title_format', 'mode', 'bar', 'gaps'

ERROR: Your command: exec rofi -modi drun run,-show drun ( There is an underline under -show drun

It also only worked when I changed bindcode to bindsym, and removed the comma. If the comma stays and bindcode is in place of bindsym no errors are shown.

**Comment by ImranVirani** [comment]
It might be distro dependant? I'm using Arctix Linux, which is arch based if that helps.

**Comment by orestisfl** [comment]
> Maybe the command needs to be quoted?

In the i3 config file? Yes, and that's how I use it in my config.

**Comment by ImranVirani** [comment]
@orestisfl That might be it, the command is not commented by default in the keycodes file, so it wasn't commented in mine. I just installed i3-gaps today.

File: etc/config.keycodes [hunk 1]
```diff
@@ -48,7 +48,7 @@ bindcode $mod+Shift+24 kill
 # start dmenu (a program launcher)
 bindcode $mod+40 exec --no-startup-id dmenu_run
 # A more modern dmenu replacement is rofi:
-# bindcode $mod+40 exec rofi -modi drun,run -show drun
+# bindcode $mod+40 exec rofi -modi drun run -show drun
```
Comment (orestisfl) [inline etc/config.keycodes]: That should work:

```suggestion
# bindcode $mod+40 exec "rofi -modi drun,run -show drun"
```

**Review by orestisfl — CHANGES_REQUESTED** [review]
Also please update this file with the same changes: https://github.com/i3/i3/blob/next/etc/config

**Comment by ImranVirani** [comment]
Updated. Thanks for the help.

**Review by orestisfl — APPROVED** [review]

**Pull request merged**